### ISSUE TITLE
Incompatible with pflag due versioning

### ISSUE THREAD

**Issue by hoshsadiq** [cornfeedhobo/pflag#23]
Seems go modules is refusing to install v2 when spf13/pflag is still on v1:

```
$ go mod tidy
go: errors parsing go.mod:
/home/hosh/workspace/myapp/go.mod:32: replace github.com/cornfeedhobo/pflag: version "v2.0.0" invalid: module contains a go.mod file, so major version must be compatible: should be v0 or v1, not v2
```

Not really sure what the solution would be besides re-tagging it as v1 (i.e. anything `<2.0.0`), but then I don't really know enough about go modules. If this is indeed the only solution, it means that this library can never reach v2 until spf13/pflag gets to v2 _if_ the aim is the keep backwards compatibility with cobra.

Anyway, currently the way to get around it is by simply using the latest version.

**Comment by cornfeedhobo** [comment]
@hoshsadiq Thank you very much for your report. I'll look into this today.

**Comment by cornfeedhobo** [comment]
@hoshsadiq It seem this is indeed a hard limit with go modules, and I wasn't aware of it either. I've delete the v2.0.0 tag, re-tagged and re-released as v1.1.0. I'll make sure to keep this in mind for the future. Thank you again for this report!
